static/app/utils/profiling/flamegraph.ts

```diff
@@ -72,7 +72,7 @@ export class Flamegraph {
       this.configSpace = new Rect(
         0,
         0,
-        configSpace ? configSpace.width : this.profile.endedAt,
```

Comment:
Are you changing this because we have a lot of space at the end of the chart? I was talking to @phacops about this - we would expect endedAt to = startedAt + duration here, which is currently not the case. I left it like that as he was doing some backend changes to fix the whole weights problem. I'm fine doing this, we just need to be aware that duration here is not something that the backend returns, but something that we are computing at the end when each profile is built